FileAPI/url/cross-global-revoke.sub.html

```diff
@@ -0,0 +1,63 @@
+<!doctype html>
+<meta charset="utf-8">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<body>
+<script>
+async_test(t => {
+  const blob_contents = 'test blob contents';
+  const blob = new Blob([blob_contents]);
+  const url = URL.createObjectURL(blob);
+  const frame = document.createElement('iframe');
+  frame.setAttribute('style', 'display:none;');
+
+  self.addEventListener('message', t.step_func(e => {
+    if (e.source != frame.contentWindow) return;
+    if (e.data == 'ready') {
+      e.source.postMessage({url: url}, '*');
+      return;
+    }
+    assert_equals(e.data, 'revoked');
+    promise_rejects(t, new TypeError, fetch(url)).then(t.step_func_done());
+  }));
+
+  frame.src = 'resources/revoke-helper.html';
+  document.body.appendChild(frame);
+}, 'It is possible to revoke same-origin blob URLs from different frames.');
+
+async_test(t => {
+  const blob_contents = 'test blob contents';
+  const blob = new Blob([blob_contents]);
+  const url = URL.createObjectURL(blob);
+  const worker = new Worker('resources/revoke-helper.js');
+  worker.onmessage = t.step_func(e => {
+    assert_equals(e.data, 'revoked');
+    promise_rejects(t, new TypeError, fetch(url)).then(t.step_func_done());
+  });
+  worker.postMessage({url: url});
+}, 'It is possible to revoke same-origin blob URLs from a different worker global.');
+
+async_test(t => {
+  const blob_contents = 'test blob contents';
+  const blob = new Blob([blob_contents]);
+  const url = URL.createObjectURL(blob);
+  const frame = document.createElement('iframe');
+  frame.setAttribute('style', 'display:none;');
+
+  self.addEventListener('message', t.step_func(e => {
+    if (e.source != frame.contentWindow) return;
+    if (e.data == 'ready') {
```

Comment:
nit: would prefer `===` to `==` everywhere, so the reader doesn't have to reason about coercion.

Reply:
done